connect/reports/renderers/pdf.py

```diff
@@ -30,8 +30,8 @@ def local_fetcher(url, root_dir=None, template_dir=None):
 
 @register('pdf')
 class PDFRenderer(Jinja2Renderer):
-    def render(self, data, output_file):
-        rendered_file = super().render(data, output_file)
+    def generate_report(self, data, output_file):
```

Comment:
I miss the test of the extension for the output_file argument as you have done for the other renderers.

Reply:
Done